actionners/actionners.go

```diff
@@ -27,18 +30,21 @@ import (
 	"github.com/falco-talon/falco-talon/internal/rules"
 	"github.com/falco-talon/falco-talon/metrics"
 	"github.com/falco-talon/falco-talon/notifiers"
+	"github.com/falco-talon/falco-talon/outputs/model"
 	"github.com/falco-talon/falco-talon/utils"
 )
 
 type Actionner struct {
 	Name                    string
 	Category                string
-	Action                  func(action *rules.Action, event *events.Event) (utils.LogLine, error)
+	Action                  func(action *rules.Action, event *events.Event) (utils.LogLine, *model.Data, error)
 	CheckParameters         func(action *rules.Action) error
 	Init                    func() error
 	Checks                  []checkActionner
 	DefaultContinue         bool
 	AllowAdditionalContexts bool
+	AllowOutput             bool
```

Comment:
Why do we have AllowOutput and RequireOutput?

Reply:
We could.